spec/mongo/bulk_write_spec.rb

```diff
@@ -2390,4 +2390,42 @@
       end
     end
   end
+
+  describe ".acknowledged?" do
```

Comment:
`#acknowledged?` since it's an instance method, `.acknowledged?` would be a class method.

Reply:
done